File: delete.py
import requests

def send_delete_request(url):
    try:
        response = requests.delete(url)
        print("ENVOI de la requete")
    #     response.raise_for_status()  # Vérifie si la réponse est un succès
        print("La requête DELETE a été envoyée avec succès.")
    except requests.exceptions.RequestException as e:
        print("Erreur lors de l'envoi de la requête DELETE:", e)

# URL de la ressource que vous souhaitez supprimer
url = 'http://localhost:8080/resource/123'

# Envoi de la requête DELETE
send_delete_request(url)

File: delete.php
<?php
    // echo 'ceci est un test';
    $url = 'http://localhost:8080/resource/123';

    // Initialiser une session cURL
    $ch = curl_init($url);
    
    // Configurer les options de la requête DELETE
    curl_setopt($ch, CURLOPT_CUSTOMREQUEST, 'DELETE');
    curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
    
    // Exécuter la requête
    $response = curl_exec($ch);
    echo $response;
    
    // Vérifier s'il y a des erreurs
    if(curl_errno($ch)){
        echo 'Erreur cURL : ' . curl_error($ch);
    }
    
    // Fermer la session cURL
    curl_close($ch);
    
    // Afficher la réponse du serveur
    echo $response;
?>